### PR TITLE
Use a Monaco Editor for the output as well

### PR DESCRIPTION
**Pull request by 333fred**
In addition to using a Monaco editor for the input, we now use one for the output, and colorize C# outputs with the built-in colorization.

### CODE REVIEW

**Comment by 333fred** [comment]
image
Any ideas on this Jan?

**Comment by jjonescz** [comment]
> Any ideas on this Jan?

Don't worry about it, automatic deployments of fork branches don't work for some reason, I wasn't yet able to figure out why.

File: src/App/Lab/Page.razor [hunk 1]
```diff
@@ -505,22 +524,30 @@
...
 
-    private string Output => selectedOutputType switch
+    private async Task UpdateOutputDisplayAsync()
     {
-        _ when compiled is null => "",
-        null => "",
-        var other => LoadOutput(other),
-    };
+        Console.WriteLine(selectedOutputType);
+        Console.WriteLine(compiled);
```
Comment (jjonescz) [inline src/App/Lab/Page.razor:530]:
```suggestion
```

File: src/App/Lab/Page.razor [hunk 2]
```diff
@@ -505,22 +524,30 @@
...
     {
         return CurrentCompiledFile?.GetOutput(type)
             ?? compiled?.GetGlobalOutput(type);
     }
 
-    private string LoadOutput(string type)
+    private async Task<string> LoadOutput(string type)
     {
         var output = GetOutput(type);
+        Console.WriteLine("Output is null: " + (output is null));
```
Comment (jjonescz) [inline src/App/Lab/Page.razor:550]:
```suggestion
```

**Review by jjonescz — APPROVED** [review]
Thanks!

**Pull request merged**